plugins/lv2/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-set(LV2_PLUGIN_PRJ_NAME "${PROJECT_NAME}_lv2")
+set(LV2_PLUGIN_PRJ_NAME "plugins_lv2")
```

Comment:
This might have side effects on the actual LV2 bundle name?

Reply:
also looking at the plugins validation in macos and windows seems all OK.